challenges/easy/who-likes-it/solutions/valentinadagata/main.py

```diff
@@ -0,0 +1,44 @@
+def likes(people):
+    num = len(people)
```

Comment:
Consider naming this variable `peopleLength` or something similar; `num` can be a bit misleading